src/towncrier/test/test_create.py

```diff
@@ -249,3 +249,35 @@ def test_create_orphan_fragment_custom_prefix(self, runner: CliRunner):
         self.assertEqual(len(change.stem), 11)
         # Check the remainder are all hex characters.
         self.assertTrue(all(c in string.hexdigits for c in change.stem[3:]))
+
+    @with_isolated_runner
+    def test_in_different_dir_with_nondefault_newsfragments_directory(self, runner):
+        """
+        Config location differs from the base directory for news file and fragments.
+
+        This is useful when multiple projects share one towncrier configuration.
```

Comment:
Maybe just focus on describing the core functionality that is tested here.


```suggestion
        When the `--dir` CLI argument is passed,
        it will create a new file in directory that is
        created by combining the `--dir` value
        with the `directory` option from the configuration
        file.
```